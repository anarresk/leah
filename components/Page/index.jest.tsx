import React from 'react';

import {Page} from './index';
import expectShallow from '../../utils/testing/expectComponent';

describe('components/Page', () => {
    it('renders as expected', () => {
        expectShallow(<Page><h1>Hello Next!</h1></Page>).toMatchSnapshot();
    });
});
