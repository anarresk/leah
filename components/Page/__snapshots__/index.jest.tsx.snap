// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Page renders as expected 1`] = `
<React.Fragment>
  <Meta />
  <Header />
  <main>
    <h1>
      Hello Next!
    </h1>
  </main>
</React.Fragment>
`;
